crates/optimism/node/src/node.rs

```diff
@@ -205,6 +207,9 @@ pub struct OpAddOns<N: FullNodeComponents> {
     pub rpc_add_ons: RpcAddOns<N, OpEthApi<N>, OpEngineValidatorBuilder>,
     /// Data availability configuration for the OP builder.
     pub da_config: OpDAConfig,
+    /// Sequencer client, configured to forward submitted transactions to sequencer of given OP
+    /// network.
+    pub sequencer_client: Option<SequencerClient>,
```

Comment:
this makes sense I think,

not quite sure yet how to configure this properly, need to check